data/recs/2015/project_multifamily_beta/Occupants.tsv

```diff
@@ -0,0 +1,27 @@
+Dependency=Geometry Building Type	Dependency=Bedrooms	Option=1	Option=2	Option=3	Option=4	Option=5	Option=6	Option=7	Option=8	Option=9	Option=10	Option=11	Option=12	Count	Weight
+Mobile Home	1.0	0.4604773753369561	0.3660734211374407	0.10121053988294497	0.0	0.0	0.07223866364265816	0.0	0.0	0.0	0.0	0.0	0.0	18.0	331607.829825
+Mobile Home	2.0	0.3359586814337728	0.3566322701292226	0.1571628739595786	0.08927566363983044	0.039216130578740925	0.003654033918528328	0.008194431851498155	0.0	0.0	0.009905914488828287	0.0	0.0	115.0	2446634.0111589995
+Mobile Home	3.0	0.1755958057043375	0.3747994026285547	0.1559955858410931	0.1427467862660943	0.07115142138798185	0.045125570764721935	0.01938400135663146	0.0	0.003043557888892462	0.0	0.0	0.012157868161692731	133.0	3547558.069917
+Mobile Home	4.0	0.2828218203129357	0.07338407968883125	0.288849978305914	0.191068992583272	0.07823105411531123	0.08564407499373593	0.0	0.0	0.0	0.0	0.0	0.0	19.0	444752.606538
+Mobile Home	5.0	0.0	0.0	0.0	1.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	0.0	1.0	16490.48254
```

Comment:
Okay to commit this as this, but for rows with sample size < 10, we should address by relaxing dependency (e.g., combine 4 and 5 bedroom samples, combine two multifamily types) or use Bayesian updating. Combining RECS 2009 and 2015 samples could also help. cc @JLReyna